examples/07-Train-traditional-ML-models-using-the-Merlin-Models-API.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>These are two of the same metrics used above, right? Shall we add a short note here to mention that they are both higher than their implicit counterparts, which means lightfm performed better? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>I wouldn't be inclined to make such a statement in a notebook aimed at being a tutorial. You generally need to be very careful when evaluating the performance of different frameworks, make sure you tuned the hyperparameters properly, make sure the comparison is fair, etc. Plus there is no signal to the user here, IMO. The performance is likely to vary greatly between these two algorithms given the size of the dataset, relationships in the dataset, etc. Possibly it is best strategy to try both models, especially given how little effort that costs (a user can use our high level API with the same dataset).</p>